Dockerfile

```diff
@@ -105,6 +105,8 @@ RUN ccache -s
 # Install doc requirements
 ADD docs/.sphinx/requirements.txt /doc-requirements.txt
 RUN pip3 install -r /doc-requirements.txt
+# Composable Kernel requires upgraded cmake
+RUN pip3 install --upgrade cmake
```

Comment:
You should pin the cmake version, so the builds are reproducible.

Reply:
Hi @bghimireamd I assume you are referring to issue resolved by this PR https://github.com/ROCmSoftwarePlatform/composable_kernel/pull/940

```
cmake_minimum_required(VERSION 3.14)
if(POLICY CMP0140)
  # policies CMP0140 not known to CMake until 3.25
  cmake_policy(SET CMP0140 NEW)
endif()
```
So with this fix, actually upgrading cmake version will not be necessary.

Reply:
@junliume 

MIOpen docker   : `rocm/miopen:ci_686727` 
MIOpen docker's cmake version : `3.16.3`
CK  develop's commit hash   : `bd09b5c5389c9f7f51459e5a38d3d7253b0c8dc0`

When building CK on MIOpen docker I still get linking error :/

Reply:
Have you rebuilt the docker? Did the cmake version get upgraded? Can you share the link to the logs?

Reply:
@illsilin I updated my comment. Sorry for confusion. When we rebuilt the docker with this PR which updates the CMake version to 3.27.5 CK builds works. It just with old version cmake even though with latest CK the build fails.

Reply:
ah, ok. false alarm.